pilot/utils/arguments.py

```diff
@@ -47,7 +47,11 @@ def get_arguments():
 
     if 'app_id' in arguments:
         if app is None:
-            app = get_app(arguments['app_id'])
+            try:
+                app = get_app(arguments['app_id'])
+            except ValueError as err:
+                print(color_red(f"Error: {err}"))
```

Comment:
If there's no app with this app id, show the message in red and exit nicely, without the stacktrace so it doesn't look like a bug.